### DISCUSSION TITLE
Use Tiptap editor as rich-text field instead of Trix

### DISCUSSION THREAD

**Discussion by yob-yob** [filamentphp/filament#1748] [Ideas]
Link to Tiptap Editor Website: https://tiptap.dev

This editor should be much more dynamic and powerful than trix. 

it also offer better customization and the ability to add more features.

**Comment by ryangjchandler** [comment]
@yob-yob I don't think we'd replace the existing Trix editor anytime soon, if at all. It should be simple enough to write a custom field for TipTap though.

**Comment by grahammtbr** [comment]
Up until the recent (Filament provided) update to Trix to use paragraph tags that made it usable for front-end content I would have strongly agreed with @yob-yob's suggestion (and I still kind of do). IMO, Trix is not a great editor without the paragraph hack, and even then its tough to extend, and you're at the mercy of Basecamp who have a long history of ignoring pull requests for it.

TipTap is awesome, however it's been [a bit a nightmare](https://github.com/ueberdosis/tiptap/issues/1515) to get working correctly with Alpine v3. It worked great with v2, but I still haven't managed to get a fully functioning custom field using TipTap and Alpine3. If anyone here is up for the challenge...
